exercises/simple-cipher/simple_cipher.py

```diff
@@ -1,5 +1,11 @@
 class Cipher(object):
-    def __init__(self):
+    def __init__(self, key):
```

Comment:
You proposed good names, but could you please set `key` to `None` by default since some test cases ignore it - https://github.com/exercism/python/blob/master/exercises/simple-cipher/simple_cipher_test.py